docs/install.rst

```diff
@@ -136,7 +152,13 @@ You also have to set where your tenant model is.
 
     TENANT_MODEL = "customers.Client" # app.Model
 
-Now run ``migrate_schemas --shared`` to create the shared apps on the ``public`` schema. Note: your database should be empty if this is the first time you're running this command.
+Now you can create your app migrations, ``customers`` in our case (otherwise Django >= 1.9 will not create its table):
```

Comment:
I suggest dropping the version 1.9 reasoning - the project only supports 1 out of 3 versions that would create the table without migrations; treat `makemigrations` as a necessary step.